frontend/src/api/instances.js

```diff
@@ -9,15 +9,15 @@ import paginateUrl from '../utils/paginateUrl.js'
 import client from './client.js'
 
 const create = async (options) => {
-    // Backend calls these flow templates, UI refers to it as flow blueprints
-    options.flowTemplateId = options.flowBlueprintId
-    delete options.flowBlueprintId
-
+    if (options.flowBlueprintId === '') {
+        delete options.flowBlueprintId
```

Comment:
Does the backend not handle receiving ''?  I think it should!

Reply:
It does handle it. I just dislike sending blank properties if they are meant to be ignored.